base/src/main/java/org/mozilla/jss/netscape/security/x509/OIDMap.java

```diff
@@ -131,43 +143,28 @@ private static void loadNamesDefault(Properties props) {
         props.put(CERT_POLICIES, "2.5.29.32");
         props.put(AUTH_KEY_IDENTIFIER, "2.5.29.35");
         props.put(SUBJ_DIR_ATTR, "2.5.29.9");
-        props.put(EXT_KEY_USAGE, "2.5.29.37");
```

Comment:
Just to confirm, so the `EXT_KEY_USAGE` doesn't need to be specified here anymore?

Reply:
The `ExtendedKeyUsageExtension` class was adding itself [here](https://github.com/dogtagpki/jss/blob/66bd995f79a5a05c78823821535f4fb4908eaea1/base/src/main/java/org/mozilla/jss/netscape/security/extensions/ExtendedKeyUsageExtension.java#L65-L71). I have not investigated there reason but it is like adding this extension whatever is the map. However, this code was not executed so I moved in the static block of the OIDMap with the `addClass()` method.